corfudb-tools/src/main/java/org/corfudb/compactor/CorfuRuntimeHelper.java

```diff
@@ -0,0 +1,140 @@
+/* *****************************************************************************
+ * Copyright (c) 2016-2019. VMware, Inc.  All rights reserved. VMware Confidential
+ * ****************************************************************************/
+package org.corfudb.compactor;
+
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.runtime.proto.service.CorfuMessage.PriorityLevel;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.CorfuRuntime.CorfuRuntimeParameters;
+import org.corfudb.runtime.CorfuRuntime.CorfuRuntimeParameters.CorfuRuntimeParametersBuilder;
+import org.corfudb.runtime.exceptions.UnreachableClusterException;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.util.List;
+
+@Slf4j
+public class CorfuRuntimeHelper {
```

Comment:
Isn't this class a copy-paste of `RuntimeParameters` class?

Reply:
No. This is more of a Helper class to create the runtime for CorfuStoreCompactorMain

Reply:
I mean, you can see, it uses the same parameters that we already have in org.corfudb.runtime.RuntimeParameters, let's use them instead of creating it second time?

Reply:
Yes, can do that and in fact remove the need of the helper class maybe.